docs/web/howto/web-in-60/static-content.rst

```diff
@@ -21,6 +21,10 @@ First, we need to import some objects:
 
     from twisted.internet import reactor
 
+- And the :api:`twisted.internet.endpoints <endpoints>` module, which gives us tools for creating listening sockets::
```

Comment:
gives us, among others, tools .... so people will not think that endpoints are only for server side :)